argo-cd/create.sh

```diff
@@ -1 +1,11 @@
-helm install argo/argo-cd --namespace argocd --values argo-cd-values.yaml --name argocd
+#!/bin/sh
+VALUES=$1
+if [ -z "${VALUES}" ]; then
+    VALUES="argo-cd"
+fi
+vfile="${VALUES}-values.yaml"
+if ! [ -e "${vfile}" ]; then
+    echo "No values file ${vfile}!" 1>&2
+    exit 2
+fi
+helm install argo/argo-cd --namespace argocd --values ${vfile} --name argocd
```

Comment:
Can we not add the needlessly complicated mechanism to pick a filename?

Reply:
But this way you can just run `create.sh` with an argument like `kueyen` to select the values file, and if you just run it with no arguments, it behaves just like it used to.